### DISCUSSION TITLE
Remove the mouse-over time and add class to ec-event?

### DISCUSSION THREAD

**Discussion by foremaple** [vkurko/calendar#351] [Q&A] [answered]
Q1: Is there a way to hide the time node added on mouse over the time grid?

![mouseover time node](https://github.com/user-attachments/assets/bda028e8-6556-42bc-88c6-a1acb7b3382c)

Q2: For ec-event node, is it possible to add a custom class to it?

**Comment by vkurko** [comment, marked as answer]
Q1: You have [pointer](https://github.com/vkurko/calendar?tab=readme-ov-file#pointer) enabled. Just disable it.

Q2: See [eventClassNames](https://github.com/vkurko/calendar?tab=readme-ov-file#eventclassnames).

**Reply by foremaple** [reply]
Thank you!
